<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="Baumplot1">
     <property name="currentIndex">
      <number>8</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Allgemein</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0" colspan="2">
          <widget class="QComboBox" name="spaufsuchenichtbegehbarursacheid"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="spaufsucheaufnahmetruppgnss">
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="spaufsucheaufnahmetruppkuerzel">
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QComboBox" name="spaufsuchenichtwaldursacheid"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="spaufsuchenichtwaldursacheidLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="spaufsuchenichtbegehbarursacheidLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Koordinaten</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <property name="horizontalSpacing">
          <number>20</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="4" column="1">
          <widget class="QLineEdit" name="istgeom_x"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Stichprobenpunkt verschoben weil:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="s_permaLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="9" column="0">
          <widget class="QLineEdit" name="istgeom_hdop"/>
         </item>
         <item row="7" column="0">
          <widget class="QLineEdit" name="istgeom_elev">
           <property name="maxLength">
            <number>32771</number>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="istgeom_sat"/>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="spaufsucheverschobenursacheid"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="s_perma"/>
         </item>
         <item row="6" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="8" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="istgeom_vdop"/>
         </item>
         <item row="4" column="0">
          <widget class="QLineEdit" name="istgeom_y">
           <property name="maxLength">
            <number>32768</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="istgeom_yError">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="istgeom_xError">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Baumplot 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLineEdit" name="azimuttransektploteins"/>
       </item>
       <item row="2" column="0">
        <widget class="QgsCollapsibleGroupBox" name="baumplot1">
         <property name="title">
          <string>Baumplot 1</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="2" column="0">
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="azimutError">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="azimut"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="distanz"/>
          </item>
          <item row="4" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="baumplotAdd">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="distanzError">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="baumplotAddError">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:red;</string>
            </property>
            <property name="text">
             <string>Ein Baum mit gleicher Distanz und Azimut existiert bereits.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="azimuttransektploteinsError">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Landmarken1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Baumplot2</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Landmarken2</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Transekt</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QComboBox" name="schutzmassnahmeid"/>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="transektstoerungursache"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Verjuengungstransekt</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>Weiserpflanzen</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="2" column="0">
        <widget class="QgsCollapsibleGroupBox" name="mGroupBox_2">
         <property name="title">
          <string>Kraut</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="4" column="0">
           <widget class="QLabel" name="weiserpflanzengoldnesselError">
            <property name="styleSheet">
             <string notr="true">color:red</string>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="weiserpflanzenbrennessel"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="krautError">
            <property name="styleSheet">
             <string notr="true">color:red</string>
            </property>
            <property name="text">
             <string>Die Summe der Anteile darf 100 % nicht überschreiten.</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLineEdit" name="weiserpflanzenheidekraut"/>
          </item>
          <item row="7" column="0">
           <widget class="QLineEdit" name="weiserpflanzenspringkraut"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="weiserpflanzenheidekrautError">
            <property name="styleSheet">
             <string notr="true">color:red</string>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="weiserpflanzengoldnessel"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="weiserpflanzenbrennesselError">
            <property name="styleSheet">
             <string notr="true">color:red</string>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="weiserpflanzenspringkrautError">
            <property name="styleSheet">
             <string notr="true">color:red</string>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QgsCollapsibleGroupBox" name="mGroupBox">
         <property name="title">
          <string>Moose</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0">
           <widget class="QLineEdit" name="weiserpflanzenmoos"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="weiserpflanzenmoosError">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color:red</string>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QgsCollapsibleGroupBox" name="mGroupBox_3">
         <property name="title">
          <string>Grass</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="krautanteil"/>
       </item>
       <item row="4" column="0">
        <widget class="QgsCollapsibleGroupBox" name="mGroupBox_4">
         <property name="title">
          <string>Farne</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>Transektinfo</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>Bestandsbeschreibung</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_12">
      <attribute name="title">
       <string>Bestockung</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_13">
      <attribute name="title">
       <string>Bodenvegetation</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_14">
      <attribute name="title">
       <string>Störung</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
